chartfx-chart/src/main/java/de/gsi/chart/plugins/TableViewer.java

```diff
@@ -503,11 +503,11 @@ private void updateEditableState() {
                 final EditableDataSet editableDataSet = (EditableDataSet) ds;
                 final EditConstraints editConstraints = editableDataSet.getEditConstraints();
 
-                if (type == ColumnType.X && editConstraints != null && !editConstraints.isXEditable()) {
+                if (type == ColumnType.X && editConstraints != null && !editConstraints.isEditable(DataSet.DIM_X)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'DataSet.DIM_X' due to existing static import 'de.gsi.dataset.DataSet.DIM_X'](https://app.codacy.com/app/GSI/chart-fx/pullRequest?prid=4477772)